### PR TITLE
Mobile layout boost

### PR DESCRIPTION
**Pull request by morekid**
Improves layout and readability for smaller and mobile screens. Quite a few changes mostly focused on:

- Content margins.
- Cards padding.
- Font sizing.
- Fixing glitches.

Also reduces spacing around the server monitoring page intro box.

### CODE REVIEW

**Comment by davidmytton** [comment]
Looks great - the changes are much better for readability on mobile.

I found an issue on the homepage on a mobile device (iPhone). When you try and swipe to scroll down, the scroll seems to get "stuck" halfway down the page. You have to try and scroll again for it actually get to the footer. The branch is deployed to test.console.dev if you want to try and reproduce it?

File: themes/console-home/layouts/tools/list.html [hunk 1]
```diff
@@ -66,7 +66,7 @@ <h2 class="title title-1">Developer tools reviews</h2>
         {{- range first 1 $.Site.Data.toolsreviewed.items -}}{{- $reviewedChecked = time (index . "Scheduled for") -}}{{- end -}}
         {{- $updated := $reviewedChecked -}}
             <p class="small">
-				<span>Last updated: {{ $updated.Format "2006-01-02" }}</span>. Read our&nbsp;<a class="link" href="/about/#selection-criteria">selection criteria</a>
+				<span>Last updated: {{ $updated.Format "2006-01-02" }}.</span>&nbsp;Read our&nbsp;<a class="link" href="/about/#selection-criteria">selection criteria</a>.
```
Comment (davidmytton) [inline themes/console-home/layouts/tools/list.html:69]: Do we need `&nbsp;` rather than just a space?

Comment (morekid) [reply themes/console-home/layouts/tools/list.html]: We shouldn't, I've refactored this to keep regular inline flow inside `span` see next commit.

**Comment by morekid** [comment]
@davidmytton the latest two commits should address both comments.

**Comment by davidmytton** [comment]
Great - all looks good.

**Pull request merged**